torch/__init__.py

```diff
@@ -243,6 +243,11 @@ def __bool__(self):
     def __int__(self):
         return self.node.int_()
 
+    # This is a hack, shouldn't be necessary.  Helps
+    # pyhpc_turbulent_kinetic_energy and vision_maskrcnn
+    def __iadd__(self, other):
+        return self + other
```

Comment:
Are you sure this isn't needed for all the other inplace ops? In fact, it seems like most the binary magic methods are missing from this SymInt shim wrapper (except for comparison ones (gt, lt) etc)

Reply:
The non inplace methods are metaprogrammed in when you load symbolic_shapes module. The inplace methods in principle would  all be needed, but they should be unnecessary and I think there is a dynamo bug

Reply:
@ezyang I meant is this bug affected all other inplace magic methods too and what should we do to fix it.

Reply:
It does but I would rather fix Dynamo to correctly do inplace logic, as it's not supposed to be necessary to define all these